src/main/java/org/example/Character/Mappers/MapFeatures.java

```diff
@@ -0,0 +1,24 @@
+package org.example.Character.Mappers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import org.example.Character.Feature;
+
+import java.util.ArrayList;
+
+import static org.example.Character.JSONMapper.MAPPER;
+
+public class MapFeatures {
+
+    public ArrayList<Feature> mapClassFeatures(String JSON){
+        try {
+            JsonNode jsonNode = MAPPER.readTree(JSON);
+            ArrayList<Feature> featureList = new ArrayList<>();
```

Comment:
List on the left side please

Reply:
Don't get it what's wrong here?

Reply:
Use List<Feature> featureList = new ArrayList<>();

Reply:
Corrected!